test/tables.jl

```diff
@@ -177,8 +181,8 @@ end
     df = DataFrame(rand(3,4))
     @test columnindex.(Ref(df), names(df)) == 1:4
     @test columnindex(df, :a) == 0
-    @test_throws ErrorException columnindex(df, 1)
-    @test_throws ErrorException columnindex(df, "x1")
+    # @test_throws ErrorException columnindex(df, 1)
```

Comment:
Also @quinnj - could you please comment why are these two tests disabled now?

Reply:
Here is a fix https://github.com/JuliaData/Tables.jl/pull/151.
It assumes that @quinnj wants to allow only `Symbol` as `name`.

Reply:
Yes, sorry for not following up on this.